<ui version="4.0" >
 <class>mdlgXYPlot</class>
 <widget class="QDialog" name="mdlgXYPlot" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>317</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>XY Plot</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox" >
   <property name="geometry" >
    <rect>
     <x>80</x>
     <y>150</y>
     <width>221</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons" >
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="Line" name="line_2" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>130</y>
     <width>291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>30</y>
     <width>291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="mlblTitle" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text" >
    <string>XY Plot:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="mrBtnXCorrDelCn2" >
   <property name="geometry" >
    <rect>
     <x>30</x>
     <y>50</y>
     <width>271</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>XCorr vs. DelCn2</string>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="mrBtnSpDelCn2" >
   <property name="geometry" >
    <rect>
     <x>30</x>
     <y>80</y>
     <width>271</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Sp vs. DelCn2</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="mrBtnXCorrSp" >
   <property name="geometry" >
    <rect>
     <x>30</x>
     <y>110</y>
     <width>271</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>XCorr vs. Sp</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>mdlgXYPlot</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>mdlgXYPlot</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
